internal/api/registry/manifests_test.go

```diff
@@ -425,8 +425,11 @@ func TestImageListManifestLifecycle(t *testing.T) {
 		deleteToken := s.GetToken(t, "repository:test1/foo:delete")
 
 		//as a setup, upload two images and render a third image that's not uploaded
+		//advance time between image generation to test {Min,Max}LayerCreatedAt
 		image1 := test.GenerateImage(test.GenerateExampleLayer(1))
+		s.Clock.Step()
 		image2 := test.GenerateImage(test.GenerateExampleLayer(2))
+		s.Clock.Step()
 		image3 := test.GenerateImage(test.GenerateExampleLayer(3))
```

Comment:
The clock steps are unnecessary since the generator functions do not inspect `s.Clock`.